src/test/java/com/puppycrawl/tools/checkstyle/utils/TokenUtilsTest.java

```diff
@@ -63,7 +67,7 @@ public void testGetIntFromInaccessibleField() throws NoSuchFieldException {
 
     @Test
     public void testTokenValueIncorrect() {
-        final Integer id = Integer.MAX_VALUE - 1;
+        final Integer id = 184;
```

Comment:
pitest is too smart, it force us to write good test that test each TokenType and request for next value after last token should be an problem. 
so we can do following:
1) create a test that iterate over all token types
2) create a static method that find max value at TokenTypes, and please use max+1 to get and exception.